src/components/Species/index.tsx

```diff
@@ -285,6 +289,8 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
   const userCanEdit = !isContributor(selectedOrganization);
   const { isMobile } = useDeviceInfo();
 
+  const BE_SORTED_FIELDS = ['scientificName', 'commonName', 'familyName', 'growthForm', 'seedStorageBehavior'];
```

Comment:
you can use this in the fields array line 297 as well.
```
  fields: [
    ...BE_SORTED_FIELDS,
    <remaining fields>
  ],
```
updating that list will be easier in one place..

Reply:
I saw that in your PR after creating this PR haha